Framework/PythonInterface/plugins/algorithms/VesuvioThickness.py

```diff
@@ -0,0 +1,137 @@
+from mantid.kernel import Direction
+from mantid.api import (PythonAlgorithm, AlgorithmFactory, ITableWorkspaceProperty)
+
+import numpy as np
+import math
+
+class VesuvioThickness(PythonAlgorithm):
+
+    _masses = None
+    _amplitudes = None
+    _transmission_guess = None
+    _thickness = None
+    _number_density = None
+    FOUR_PI = (math.pi * 4)
+
+    def summary(self):
+        return "Produces the sample density for vesuvio based on sample transmission and composite masses"
+
+    def category(self):
+        return "Inelastic\\Indirect\\Vesuvio"
+
+    def PyInit(self):
+        self.declareProperty("Masses", "",
```

Comment:
We should use a `FloatArrayProperty` for both the `Masses` & `Amplitudes` properties - http://www.mantidproject.org/Declaring_Array_Properties

The Python command will then accept a standard Python list, numpy array for comma-separated string as the input rather than forcing a user to convert to a string.